.github/workflows/trivy.yml

```diff
@@ -0,0 +1,35 @@
+# This workflow performs trivy check of docker config files.
+
+name: Trivy
+
+on:
+  pull_request:
+    paths:
+      - 'devops/containers/**'
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Trivy
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          sparse-checkout: devops
+
+      # There is a github action, but for some reason it ignores ignore-file.
+      - name: Install Trivy
+        run: |
+          curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/main/contrib/install.sh | sh
+          ./bin/trivy --version
+
+      - name: Run Trivy vulnerability scanner
+        run: ./bin/trivy config --format json --output trivy-report.json --ignorefile=devops/.trivyignore.yaml devops/containers
+
+      - name: Upload report artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: trivy-report
+          path: trivy-report.json
+          retention-days: 3
```

Comment:
is 3 days enough?

Reply:
Since this check takes like ~10s, I think 3 days is even too much :) you can just re-run it if needed. Anyway, what it should be in your opinion?

Reply:
i guess it depends on how do we expect the results to be used? is it just checking it when the workflow fails and not really after that? if so this is fine

Reply:
> i guess it depends on how do we expect the results to be used? is it just checking it when the workflow fails and not really after that? if so this is fine

The main use case is to upload them as an evidence of the scan.

@KornevNikita, is it correct to assume that any new issue will fail this workflow? If so, then 3 days is more than enough to grab results which are expected to be always good.

Reply:
It's good that Nick asked, turned out we need to set "exit-code" arg to get a non-zero result in case of fail - updated [0fdba39](https://github.com/intel/llvm/pull/16840/commits/0fdba39d5397c62aab75a22eb4c3b3618aa74520)

Reply:
😎

Reply:
I bet he found it out a day or two ago when hunting zombies :D